docs/introduction.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Introduction
+icon: crystal-ball
+---
+
+## What is Connect?
+**Connect** is an in-protocol oracle that securely delivers data to blockchain applications via their own validator set. This documentation will guide you through using Connect; whether you're a **validator** or **developer**.
+
+<CardGroup cols={2}>
+    <Card title="Validator Setup" icon="circle-nodes" href="/validators/quickstart">
+        Learn how to setup and run Connect with your infrastructure.
+    </Card>
+    <Card title="Using the API" icon="laptop-code" href="/using-connect/high-level">
+        Learn how to use the Connect API to access data.
+    </Card>
+</CardGroup>
+
+### Features
+
+#### Security <Icon icon="shield-halved"/>
+Connect leverages the chain’s security, giving the fastest updates possible, and removing the requirements for any 3rd party systems.
+
+#### Performance <Icon icon="gauge"/>
+Connect can support over 2000 currency pairs and price feeds, allowing the launch of thousands of permissionless on-chain markets.
+
+#### Support <Icon icon="headset"/>
+Connect comes with a 1-day SLAs for adding new feeds, and 24/7 on-call support and maintenance by the Skip team.
+
+#### UX <Icon icon="user"/>
+By leveraging new advancements in consensus like vote extensions & ABCI++, Connect guarantees a millisecond-fresh oracle update every block, allowing applications to build without sacrificing UX for safety.
+
+
+## Ready to Connect?
+
+If you're interested in using Connect, please check out our [website](https://skip.build) and reach out to us to get started.
```

Comment:
Small nit: can we have paragraphs formatted consistently after headers? 

Sometimes they are with no extra newline, sometimes they have a newline between them and the header.  I think the newline after header is nicer.

Reply:
[newline after headings](https://github.com/skip-mev/slinky/pull/604/commits/834eb8a9494d6113cee16ef870c4ec81db9e5e99)